--- origsrc/rasqal-0.9.25/docs/Makefile.am	2011-01-26 20:29:18.000000000 -0600
+++ src/rasqal-0.9.25/docs/Makefile.am	2011-06-12 01:44:07.391661800 -0500
@@ -120,10 +120,10 @@ $(top_builddir)/src/rasqal.h:
 README: README.html
 	$(LYNX) $(LYNXARGS) $< > $@
 
-rasqal-fake.i: $(top_srcdir)/src/rasqal.h $(srcdir)/../scripts/fix-gtkdoc-header.pl
-	$(PERL) $(srcdir)/../scripts/fix-gtkdoc-header.pl < $< > $@
-
 rasqal-changes.xml: $(srcdir)/rasqal-changes.tsv $(srcdir)/../scripts/process-changes.pl
 	$(PERL) $(srcdir)/../scripts/process-changes.pl --package rasqal --docbook-xml $@ "Rasqal" $(srcdir)/rasqal-changes.tsv
 
 endif
+
+$(srcdir)/rasqal-fake.i: $(top_builddir)/src/rasqal.h $(srcdir)/../scripts/fix-gtkdoc-header.pl
+	$(PERL) $(srcdir)/../scripts/fix-gtkdoc-header.pl < $< > $@
