--- origsrc/rasqal-0.9.32/utils/Makefile.am	2014-02-23 09:22:04.000000000 -0600
+++ src/rasqal-0.9.32/utils/Makefile.am	2014-05-19 17:03:38.459105600 -0500
@@ -52,7 +52,7 @@ rasqalcmdline.h
 
 roqet_SOURCES = roqet.c
 roqet_CPPFLAGS = $(AM_CPPFLAGS)
-roqet_LDADD = $(top_builddir)/src/librasqal.la librasqalcmdline.la
+roqet_LDADD =  librasqalcmdline.la $(top_builddir)/src/librasqal.la
 roqet_DEPENDENCIES = $(top_builddir)/src/librasqal.la librasqalcmdline.la
 roqet_LDFLAGS = @RAPTOR2_LIBS@
 if GETOPT
@@ -61,7 +61,7 @@ endif
 
 check_query_SOURCES = check_query.c
 check_query_CPPFLAGS = $(AM_CPPFLAGS)
-check_query_LDADD = $(top_builddir)/src/librasqal.la librasqalcmdline.la
+check_query_LDADD = librasqalcmdline.la $(top_builddir)/src/librasqal.la
 check_query_DEPENDENCIES = $(top_builddir)/src/librasqal.la librasqalcmdline.la
 check_query_LDFLAGS = @RAPTOR2_LIBS@
 if GETOPT
